.github/workflows/deployment-master.yml

```
@@ -1,46 +1,40 @@
-# name: Deploy the application
-# on:
-#   push:
-#     branches:
-#       - 'master'
-# env:
-#   SPORT: golf
+name: Deploy StyleSheets
+on:
+  push:
+    branches:
+      - 'master'
+env:
+  PROD_STATIC_DIST: ${{ secrets.PROD_STATIC_DIST }}
+ 
+jobs:
+  deploy:
+    name: Deploy Stylesheets
+    runs-on: ubuntu-latest
 
-# jobs:
-#   deploy:
-#     name: Deploy job
-#     runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v1
 
-#     strategy:
-#       matrix:
-#         node-version: [14.15.x]
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-1
+          role-to-assume: 'arn:aws:iam::682619735097:role/frs-prod-cicd'
+          role-duration-seconds: 1200
 
-#     steps:
-#       - name: Checkout the repository
-#         uses: actions/checkout@v1
-
-#       - name: Use Node.js ${{ matrix.node-version }}
-#         uses: actions/setup-node@v1
-#         with:
-#           node-version: ${{ matrix.node-version }}
-#           registry-url: https://registry.npmjs.org/
-
-#       - name: npm install and build
-#         run: |
-#           npm ci
-#           npm run build
-#         env:
-#           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
-
-#       - name: Configure AWS Credentials
-#         uses: aws-actions/configure-aws-credentials@v1
-#         with:
-#           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-#           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-#           aws-region: eu-west-1
-#           role-to-assume: 'arn:aws:iam::460928140709:role/frs-dev-cicd'
-#           role-duration-seconds: 1200
-
-#       - name: Deploy to AWS
-#         run: |
-#           aws s3 sync dist s3://imgarena-apps-static-assets-frs-dev/${SPORT}/latest --acl private --delete
+      - name: Invalidate Cache Stylesheets
+        run: |
+          export INV_ID=`aws cloudfront create-invalidation --distribution-id $PROD_STATIC_DIST --paths "/stylesheets/*" | jq -r '.Invalidation.Id'`
+          sleep 120
+          export INV_STATUS=`aws cloudfront get-invalidation --id $INV_ID --distribution-id $PROD_STATIC_DIST |  jq -r '.Invalidation.Status'`
+          if [ "$INV_STATUS" == "Completed" ]; then
+            echo "Invalidation of Stylesheets Cache. Status=$INV_STATUS"
+          else
+            echo "ERROR: Invalidation of Cache has not finished successfully. STATUS=$INV_STATUS"
+            exit 1
+          fi
+      - name: Deploy Stylesheets
+        run: |
+          aws s3 sync stylesheets s3://imgarena-static/stylesheets --acl private --delete --cache-control max-age=86400 
```

I think the cache-control needs an additional value based on what we use in the event centers
```suggestion
          aws s3 sync stylesheets s3://imgarena-static/stylesheets --acl private --delete --cache-control "public, max-age=86400"
```

Event centre link to the github actions https://github.com/IMGARENA/ufc-event-centre/blob/master/.github/workflows/deployment-master.yml#L41